onnxmltools/convert/common/_topology.py

```diff
@@ -648,6 +649,19 @@ def convert_topology(topology, model_name, doc_string, targeted_onnx):
                 else:
                     other_outputs[variable.raw_name] = variable
 
+    # Check input and output naming convention
+    for name in topology.raw_model.input_names:
+        input_name = name.replace('_', '')
+        if input_name[0].isdigit() or (not input_name.isalnum()):
```

Comment:
is underline included in isalnum?

Reply:
@stevenlix , why do we replace `_` with space? It's a C-stype name.

Reply:
There is no change on any names. Here is to check whether the names are compliant with ONNX spec. If it's not, a warning is generated.

Reply:
I got your point. So could you please merge the two new loops with the following ones (removing `break` so that all names can be examined)?

Reply:
I think there is no need to generate warning for each name, so I just generate one warning as long as I detect an incompliant name. Users should check the model input/output names once they see the warning.

Reply:
most likely, names are either all compliant or all incompliant

Reply:
But the names are generated by us..

Reply:
The input and output names are coming from original model that to be converted, aren't they?

Reply:
From the end user view, it would be better to print more names which are incompliant.